documentation/01_tutorial/04-3-groundwork.html.md

```diff
@@ -72,7 +73,12 @@ First up, let's create a simple menu. Eventually, we'll want a fancy `MenuState`
 
 	`screenCenter()` is a function which takes an object and centers it on the screen either horizontally, vertically or (by default) both. By calling it on our button, the button will be placed in the center of the screen.
 
-5. Next, we need to make sure that the game actually starts with our `MenuState`. Open `Main.hx` and replace the `PlayState` reference with `MenuState`.
+5. Next, we need to make sure that the game actually starts with our `MenuState`. Open `Main.hx` and replace the `PlayState` reference with `MenuState`. You have to add the reference to the MenuState file using on the top of the `Main.hx`
```

Comment:
No you don't, modules from the same package are imported automatically.

Reply:
Good to know 👍